0xPolygon/polygon-edge
# Start checkpoint process on every epoch start

# Description

Fixes EDGE-457

# Changes include


# Checklist


## Testing


### Manual tests

I tested this code manually with logging all blocks after fetching them in StartNewCheckpoint (checkpoint.go)
Steps:
1. Add logs
2. Start network with --bridge and --epoch-size 10 flag 
3. On every 10th block, all blocks from the previous epoch should get logged
